crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -2287,6 +2293,50 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn does_not_duplicate_global_registration_2311() {
```

Comment:
I think by mentioning the issue number you can easily convey more information why the test is there, and why this contract exists in the first place. I could've written a detailed comment about what behaviour this test is testing and why it exists, which would just be restating things from the issue, or just tag the issue number so somebody can look up that issue and hopefully understand why we test that behaviour, since it's not really well documented anywhere else. And usually when closing a bug I think it's good to have a test that verifies the fix, and without tagging the issue you kind of lose context.

So I'd prefer to keep the issue number in, but if you're sure, I can remove it.

Reply:
Yeah, now I get what you mean. Fixed in 191309c711.